controller_manager/src/controller_manager.cpp

```diff
@@ -39,10 +51,36 @@ ControllerManager::ControllerManager(
   // add pluginlib loader by default
   loaders_({std::make_shared<ControllerLoaderPluginlib>()})
 {
+  using namespace std::placeholders;
+  list_controllers_service_ = create_service<controller_manager_msgs::srv::ListControllers>(
+    "list_controllers", std::bind(
```

Comment:
I have some concerns about this.

Right now the advertised services are not within any namespace. They have global names such as:
```
/list_controller_types
/list_controllers
/load_controller
/reload_controller_libraries
/switch_controller
```

I believe they should be within this node's name, otherwise if we had more than one controller manager on the same ROS2 network there'd be a clash.

Any thoughts about this? @bmagyar @Karsten1987

Reply:
yes, they should definitely be namespaced.

Reply:
+1 for namespacing